ckanext/archiver/tasks.py

```diff
@@ -356,9 +357,10 @@ def _save(status_id, exception, resource, url_redirected_to=None,
         'site_url': config.get('ckan.site_url_internally') or config['ckan.site_url'],
         'cache_url_root': config.get('ckanext-archiver.cache_url_root'),
         'previous': Archival.get_for_resource(resource_id)
-        }
+    }
     try:
         download_result = download(context, resource)
+        e = {'args': ''}
```

Comment:
why is this needed?

Reply:
I think it's possible to have a non-OK status code without an actual exception, which would result in an error at line 383.